packages/dds/tree/src/test/shared-tree/undo.spec.ts

```diff
@@ -28,6 +28,7 @@ const testCases: {
 	initialState: JsonCompatible[];
 	editedState: JsonCompatible[];
 	undoState?: JsonCompatible[];
+	mergeState?: JsonCompatible[];
```

Comment:
Would nice to add doc comments on each member here. 🙏

Reply:
Some prayers are answered. Some are not 😶‍🌫️ .